docs/DeveloperGuide.md

```diff
@@ -133,85 +134,50 @@ Classes used by multiple components are in the `seedu.patientbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
-### \[Proposed\] Undo/redo feature
+### 1. Done feature
+The mark done feature allows users to mark a specific appointment in the address book as done using `d/` tag and `t/` tag to specify the appointment at a certain time slot.
+#### 1.1 Implementation
+Command: `done d/DATE t/TIME`
 
-#### Proposed Implementation
+Example Usage: 
+* `done d/Monday t/9am`
+* `done d/12-12-2020 t/12pm`
 
-The proposed undo/redo mechanism is facilitated by `VersionedPatientBook`. It extends `PatientBook` with an undo/redo history, stored internally as an `patientBookStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+The Mark Appointments as Done Feature is facilitated by the `DoneCommand`, which extends the abstract class `Command`, and
```

Comment:
Hmm perhaps the phrase "mark appointments" and "feature" should be in lower case?